.circleci/config.yml

```
@@ -38,12 +38,17 @@ commands:
           name: "Generate initial checksum manifests"
           command: |
             mkdir .manifests
-            scripts/generate-manifest.js .manifests/native_code '^Podfile' '^Makefile' '^Gemfile' '^emission/' '^Artsy'
+            scripts/generate-manifest.js .manifests/native_code '^Podfile' '^Makefile' '^Gemfile' '^emission/' '^Artsy' '^patches/react-native-*'
```

these strings are regexes so the `*` in `'^patches/react-native-*'` means "any number of dashes". You probably meant `'^patches/react-native-.*'`, although the current version will still match all file name that begin with `react-native`

We could simplify as follows though

```suggestion
            scripts/generate-manifest.js .manifests/native_code '^Podfile' '^Makefile' '^Gemfile' '^emission/' '^Artsy' '^patches/react-native'
```

---

Ahhhh yes you're right, good point 👍